eternalcore-paper/src/main/java/com/eteranlcode/containers/AdditionalContainerPaper.java

```diff
@@ -0,0 +1,33 @@
+package com.eteranlcode.containers;
+
+import io.papermc.lib.PaperLib;
+import io.papermc.lib.environments.Environment;
+import org.bukkit.ChatColor;
+import org.bukkit.entity.Player;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.logging.Logger;
+
+public final class AdditionalContainerPaper {
+
+    private static final Environment environment = PaperLib.getEnvironment();
```

Comment:
Zmienna `enviroment` jest używana tylko raz, więc po co ją tworzyć, lepiej bezpośrednio użyć `PaperLib#getEnviroment()`

Reply:
to jest tylko moja sugestia, także jeżeli wolisz aby kod był "czystszy" kliknij "Resolve conversation"